scripts/filename.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+
+function fileName(name, checkAvail = true) {
+    if (name !== undefined) {
+        const fileName = `src/${name}.ts`;
+        if (!checkAvail) {
```

Comment:
explicit boolean check, e.g. `checkAvail === false`?

Reply:
Sure, we can do that.